virttest/data_dir.py

```diff
@@ -194,6 +214,8 @@ def get_deps_dir(target=None):
 def get_tmp_dir(public=True):
     """
     Get the most appropriate tmp dir location.
+    For heavy usages like test images or requiring direct access
```

Comment:
It is better if the main docstring statement is separated from this explanation by an empty line (i.e. empty line after "Get the most appropriate tmp dir location..").